client/src/components/PositionTable.js

```diff
@@ -57,7 +56,8 @@ const PositionTable = props => {
 
 PositionTable.propTypes = {
   // query variables for positions, when query & pagination wanted:
-  queryParams: PropTypes.object
+  queryParams: PropTypes.object,
+  linkToComp: PropTypes.func.isRequired
```

Comment:
PositionTable is used in 9 places and in 7 of them the value of `linkToComp` prop is `LinkTo`. So why not provide a default prop for that?

Reply:
This is due to avoid circular dependency, we can't import `LinkTo` in this file.